src/Pos/CompileConfig/Type.hs

```diff
@@ -42,6 +40,8 @@ data CompileConfig = CompileConfig
       -- ^ VSS certificates min timeout to live (number of epochs)
     , ccLightDlgConfirmationTimeout   :: !Int
       -- ^ Timeout for holding light psks confirmations
+    , ccDlgCacheParam                 :: !Int
```

Comment:
No, not bytes. Moreover, we usually don't use types like `Byte` in `CompileConfig`, we do necessary conversions in `Constants.hs`.